tests/cases/fourslash/tripleSlashRefPathCompletionExtensionsAllowJSFalse.ts

```diff
@@ -0,0 +1,31 @@
+/// <reference path='fourslash.ts' />
+
+// Should give completions for relative references to ts files when allowJs is false
```

Comment:
For references, we have:
- tripleSlashRefPathCompletionAllowJSFalse.ts
- tripleSlashRefPathCompletionAllowJSTrue.ts

For imports, we have:
- completionForStringLiteralRelativeImportAllowJSFalse.ts
- completionForStringLiteralRelativeImportAllowJSTrue.ts

(latter two renamed in the latest commit).